spec/features/visitor_creates_an_account_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+describe User, type: :feature do 
+  describe "A visitor" do 
+    it "can register as a user with correct field inputs" do 
+      visit root_path
+      click_on "Create Account"
+      expect(current_path).to eq(new_user_path)
+
+      fill_in "First name", with: "First Name"
+      fill_in "Last name", with: "Last Name"
+      fill_in "Email", with: "Email"
+      fill_in "Phone number", with: "Phone"
+      fill_in "Password", with: "Password"
+      
+      click_on "Sign Up"
+
+      expect(page).not_to have_content("Create Account")
+      expect(page).to have_content("First Name")
+      expect(page).to have_content("Last Name")
+      expect(page).to have_content("Email")
+      expect(page).to have_content("Phone")
+    end
+
+    it "cannot register as a user without correct field inputs" do 
+      visit root_path
+      click_on "Create Account" 
+      expect(current_path).to eq(new_user_path)
+
+      fill_in "First name", with: "First Name"
+      fill_in "Last name", with: "Last Name"
+      fill_in "Email", with: "Email"
+
+      click_on "Sign Up"
+      
+      expect(current_path).to eq(new_user_path)
+      expect(page).to have_content("you done fucked up")
```

Comment:
@njgheorghita it would be suuuper helpful to know *why* the user's input is invalid.  Can we check for something like... `expect(page).to have_content("Phone number can't be blank")` 
What do you think?